File: en/jruby/Car.java
// Car.java
public class Car {
  private String brand, color, model;
  private int tank;

  public Car(String brand, String color, String model, int tank) {
    this.brand = brand;
    this.color = color;
    this.model = model;
    this.tank  = tank;
  }

  public String toString() {
    return "brand: " + this.brand + "\n" +
           "color: " + this.color + "\n" +
           "model: " + this.model + "\n" +
           "tank:  " + this.tank;
  }
}

File: en/docs/yard/car.rb
# This is the base class for *all* the cars we are going to
# create in our program. From it we create cars
# of specific _brands_.
#
# Author:: Eustáquio 'TaQ' Rangel
# Licence:: GPL
class Car
  attr_reader :brand, :model, :tank
  attr_accessor :color

  # Required parameters to create the car.
  # Do not forget that every car will have the costs of:
  # * Registration tax
  # * Mandatory insurance
  # * Insurance
  # * Maintenance
  def initialize(brand,model,color,tank)
    @brand = brand
    @model = model
    @color = color
    @tank  = tank
  end

  # Converts the car into a more readable representation
  def to_s
    "Brand:#{@brand} Model:#{@model} Color:#{@color} Tank:#{@tank}"
  end
end
